enterprise/models.py

```diff
@@ -242,8 +263,12 @@ class EnterpriseCustomerUser(TimeStampedModel):
     )
     user_id = models.PositiveIntegerField(null=False, blank=False)
 
+    active = models.BooleanField(default=True)
```

Comment:
I think it's the right move to include this only on the EnterpriseCustomerUser and not the PendingEnterpriseCustomerUser. For the pECU, we don't really have enough data to make it worthwhile to keep the record around - the user has never actually interacted with the system.